### ISSUE TITLE
Support AWS session credentials

### ISSUE THREAD

**Issue by skeggse** [buchgr/bazel-remote#634]
We use session credentials from the result of an `sts:AssumeRole` call (via the [`configure-aws-credentials`](https://github.com/aws-actions/configure-aws-credentials) action). These credentials take three parts rather than the two of static credentials: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN`. `bazel-remote` does not seem to consume the `AWS_SESSION_TOKEN` environment variable, so it tries to interpret these credentials as static credentials:
```
WARNING: Remote Cache: io.grpc.StatusRuntimeException: UNKNOWN: The AWS Access Key Id you provided does not exist in our records.
```

Ideally, this common form of expressing AWS credentials would be supported by `bazel-remote`.

References:
* seems very related to #490, but that never concerned itself with `AWS_SESSION_TOKEN` - just with the profile option

There's probably a workaround for this by using a configuration file and a profile flag, but

**Comment by pauldraper** [comment]
A workaround that should work: https://github.com/buchgr/bazel-remote/pull/493#issuecomment-984803190

**Comment by mostynb** [comment]
This was implemented in #795.
